providers/mock.go

```diff
@@ -117,7 +117,7 @@ func (s *mockProviderService) Init(running *RunningProvider) {
 	s.initialized = true
 
 	rt := s.coordinator.NewRuntime()
-	rt.schema.loadAllSchemas()
-	rt.schema.refresh()
+	rt.schema.unsafeLoadAll()
```

Comment:
i think these calls may not be needed. `newRuntime` already calls them:

```
func (c *coordinator) newRuntime(isEphemeral bool) *Runtime {
	res := &Runtime{
		coordinator:     c,
		providers:       map[string]*ConnectedProvider{},
		schema:          newExtensibleSchema(),
		Recording:       NullRecording{},
		shutdownTimeout: defaultShutdownTimeout,
		isEphemeral:     isEphemeral,
	}
	res.schema.runtime = res

	// TODO: do this dynamically in the future
	res.schema.unsafeLoadAll()
	// TODO: this step too shouild be optional only, even when loading all.
	// It is executed when the we connect via a provider, so doing it here is
	// overkill.
	res.schema.unsafeRefresh()
```

Reply:
You're right, I'll comment them out and add a comment that once we remove unsafeLoadAll from newRuntime we need to add it here.